src/TextInput/TextInput.tsx

```diff
@@ -158,12 +157,13 @@ const Content = styled.div<IInputOutline>`
 
 const Input = styled.input<IInput>`
   border: none;
+  background-color: transparent;
   color: ${({ error }) => theme.colors[`${error ? 'error' : 'secondary'}`]};
   font-size: 16px;
   width: 100%;
   outline: none;
-  cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};
-  opacity: ${({ disabled }) => (disabled ? '0.5' : '1')};
+  cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'initial')};
+  padding: ${({ outlined }) => (outlined ? '17px 14px' : '1px 2px')};
```

Comment:
It was `16px 12px` but the input has some initial padding and I didn't go further to see if it would be possible to remove this additional padding

Reply:
Do you have a screenshot? does this maintain the 58px height of the input? Doesn't seem you have changed the height of it

Reply:
Yes, those value keep everything the same height as before so `58.5px`, not sure why there is the `.5` tho